README.md

```diff
@@ -1,2 +1 @@
-# quiz-review
-commenting and review
+# quiz
```

Comment:
what are the plans for ReadMe ?

Reply:
for reference : https://github.com/OfficeDev/microsoft-teams-apps-survey/blob/main/README.md

Reply:
Documenting is not under our scope of work.

Reply:
Documentation is not under our scope of work